### PR TITLE
Adding config-parser, branch name, and job name

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This installs the new config-parser and uses it.  It also adds support for specifying branch and job name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+ - Checkout a branch
+ - Create required directories
+ - Create required environment variables
+ - Parse a screwdriver.yaml
+ - Execute parsed commands
+
+## Usage
+
+```bash
+$ /opt/job-tools/launch ${git_org} ${git_repo} ${git_branch} ${job_name}
```
Comment (d2lam) [inline README.md]: Do you want to add that git_branch and job_name are optional and what their default values would be?

Comment (stjohnjohnson) [inline README.md]: Good catch.

**Comment by nkatzman** [comment]
👍

**Comment by d2lam** [comment]
👍

**Pull request merged**